src/components/map/MapLayersControl.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+import { IoMdMap } from 'react-icons/io';
+import { RadioButton } from '../controls';
+import * as s from './mapLayersControl.scss';
+
+interface INodeWindowProps {
+}
+
+interface IMapLayersControlState {
+    selectedOption: string;
+}
+
+@observer
+export default class MapLayersControl extends React.Component
+<INodeWindowProps, IMapLayersControlState> {
+    constructor (props: any) {
+        super(props);
+        this.state = {
+            selectedOption: 'kartta',
```

Comment:
I would like this to be an enum instead of hard-coded strings.
No need to put the enum in the enum folder, the enum could well be specified locally in this file since it is only used locally.